docs/adrs/0011-nimbus-web-glean-application-structure.md

```diff
@@ -0,0 +1,122 @@
+
+## Decision Drivers
+
+* [Probe-scraper repository list](https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml) developer burden
+* Ease of data collation
+  * How easy is it to collate data for an application?
+  * How easy is it to collate data across applications?
+* Glean Python SDK architecture
+
+## Considered Options
+
+* One Glean app ID
+* One Glean app ID per domain
+* One glean app ID per application
+
+## Decision Outcome
+
+Chosen option: "One Glean app ID per product domain", because it has the most advantages and fewest disadvantages _(see below)_. 
+Option "One Glean app ID" is also viable, but it could come with additional disadvantages in the form of conflicting data retention policies across product domains.
+Option "One glean app ID per application"'s disadvantages are simply too detrimental to the project for the option to be viable.
+
+### Positive Consequences
+
+* Its usage will work with the existing Glean Python SDK architecture
+* Collating data for one app and across apps is straightforward
+* Probe-scraper repository list will have overall fewer listings for Nimbus web applications than the current model
+* Sidecar deployment model remains viable within Pocket's architecture
+* Data retention can be set differently for each product domain
+
+### Negative Consequences
+
+* Jetstream will require changes in order to support this choice
+* We are breaking away from our historical standard of Glean practices
+* We may have to make more intensive changes to Experimenter
+
+## Pros and Cons of the Options
+
+### One Glean app ID
+
+With this option, we would have one Glean application ID that is shared across all instances of Cirrus.
+All applications using Cirrus would share a single Glean application ID, allowing for an overall lower maintenance burden for probe-scraper and easier collation of data across applications.
+One downside of this solution is that data from multiple product domains will exist within the same table structure.
+
+* Good, because it has the smallest footprint in the probe-scraper repository of the listed options.
+* Good, because the Glean SDK supports this particular use case.
+* Good, because this model supports Pocket's architecture.
+* Bad, because data retention cannot be handled separately for each application or product domain.
+* Bad, because it breaks away from our existing Glean usage practices.
+* Bad, because it requires changes to Jetstream.
+  * At this time, the nature and timeline of these changes is unclear.
+* Bad, because it could require more changes to Experimenter than we already expect.
+  * While we will still need to add new applications, it is also possible we will need to adjust how applications are handled in Experimenter.
+
+### One Glean app ID per product domain
+
+In this option, we would have one Glean application ID that is shared across all applications within a given product domain.
```

Comment:
Ideally, the Experimenter API would also expose this identifier/application since this could be used by Jetstream to generate more cost efficient queries.

Reply:
Interesting.  Then we've got two identifiers, a broader domain id, and a specific app id.  Both for telemetry and targeting.  Probably some nuance there to sort out.

Reply:
Yep, I'll add that as a note here and in the single-glean-app option